app/book.py

```diff
@@ -0,0 +1,22 @@
+class Book:
+    def __init__(self, title: str, content: str) -> None:
+        self.title = title
+        self.content = content
+
+    def display(self, display_type: str) -> None:
```

Comment:
I think it is better to make an abstract class display and make 2 classes console and reverse that inherit from it